com.unity.render-pipelines.high-definition/Documentation~/SGNode-Raytracing-Quality.md

```diff
@@ -0,0 +1,17 @@
+# Raytracing Quality Node
+
+The Raytracing Quality Node allows you to provide a fast implementation of your Shader Graph to be use in GPU heavy raytraced effect to tradeoff accuracy for speed: [Ray-Traced Reflections](Ray-Traced-Reflections.md), [Ray-Traced-Global-Illumination](Ray-Traced-Global-Illumination.md) and [Ray-Traced-Subsurface-Scattering](Ray-Traced-Subsurface-Scattering.md).
```

Comment:
The Raytracing Quality Node allows you to provide a fast implementation of your Shader Graph to use in GPU heavy ray-traced effects to tradeoff accuracy for speed. The GPU heavy effects are:
* [Ray-Traced Reflections](Ray-Traced-Reflections.md).
* [Ray-Traced-Global-Illumination](Ray-Traced-Global-Illumination.md).
* [Ray-Traced-Subsurface-Scattering](Ray-Traced-Subsurface-Scattering.md).